### ISSUE TITLE
[Bug] Gallery view -- no thumbnails, font too large

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#2399]
In [gallery view on Blacklight](https://collections-uat.library.yale.edu/catalog?f%5Bcollection_title_ssi%5D%5B%5D=Lindbergh+picture+collection+%28MS+325B%29&f%5Brepository_ssi%5D%5B%5D=Manuscripts+and+Archives&view=gallery): 

- no thumbnails appear for digital objects 
- search results font size is larger than rest of page

Screenshot: 
![Image](https://user-images.githubusercontent.com/98423494/220771252-4f581fa7-3721-4b4c-af27-adfefa8c7e28.png)

We want thumbnails and a font size that matches the rest of the site. 

This occurs across TEST, UAT, and PROD. Same errors on mac/Chrome, mac/Safari, and mac/Firefox.

**Acceptance:**

- [ ] Gallery view search results display thumbnails
- [ ] Gallery view search results font size matches rest of page

**Comment by MaggieZhaoYale** [comment]
![Image](https://user-images.githubusercontent.com/46331329/221030580-02da2168-e8c7-4cf7-b2ff-ac320ef37490.png)

**Comment by mikeapp** [comment]
I don't think that is impacting the display.    The CSS in grid view appears to be using:

` -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";`

In the list view this is overridden with `Mallory-Medium, Arial, Helvetica, sans-serif;` and the correct size.

**Comment by jpengst** [comment]
Heres what the console says in grid display:  


![Image](https://user-images.githubusercontent.com/24666568/221244840-55488a25-6204-4442-8305-f36e0a171e5a.png)

**Comment by martinlovell** [comment]
https://github.com/yalelibrary/yul-dc-blacklight/pull/875 (merged)

**Comment by martinlovell** [comment]
v1.60.62

**Comment by K8Sewell** [comment]
Looks like we're back to normal on Test.  Will promote to UAT.



![Image](https://user-images.githubusercontent.com/36549923/221637294-043f2f8f-f098-4490-92b7-fefd3b04d6f7.png)

**Comment by sshetenhelm** [comment]
Yes! Looks like this is back to normal, thank you!
